refinery/core/models.py

```diff
@@ -1740,6 +1744,26 @@ def attach_outputs_downloads(self):
                     "No file found for '%s' in download '%s' ('%s')",
                     analysis_result.file_store_uuid, self.name, self.uuid)
 
+    def terminate_file_import_tasks(self):
+        """
+        Gathers all UUIDs of FileStoreItems used as inputs for the Analysis,
+        and trys to terminate their import_file tasks if possible
```

Comment:
Gathers all UUIDs of FileStoreItems used as inputs for the Analysis, and try to terminate their import_file tasks